cmd/cored/disable_mockhsm.go

```diff
@@ -1,22 +1,17 @@
-//+build prod
+//+build disable_mockhsm
```

Comment:
I'm not super fond of the `enable_foo` `disable_bar` naming scheme, because “enable” doesn't contribute any information. For the positive ones I'd rather call it just `foo`. That implies we should find an alternative to “disable” for the negative ones, so how about `no_` as a prefix? e.g.

```
no_loopback_authn
no_mockhsm
no_reset
plain_http
```